models/issues/issue.go

```diff
@@ -596,6 +596,8 @@ func IsUserParticipantsOfIssue(ctx context.Context, user *user_model.User, issue
 type DependencyInfo struct {
 	Issue                 `xorm:"extends"`
```

Comment:
an issue/pull that depend on issue_id or is blocked by issue_id. the exact usage is determined by the function using this type ...